### PR TITLE
Release 4.17.0

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.17.0 (2021-06-15)
+
+- Updated the SKY UX icon font version to 4.0.0-beta.3. [#238](https://github.com/blackbaud/skyux-theme/pull/238)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Possibly a dumb question, but do we really need the word `font` here? I'm not sure how it applies. Are our icons really a font? Or is it just the `SKY UX icon version`?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md:3]: I was just following the pattern set on a prior release. We don't necessarily need it though. If you think its unnecessary, we can strip it out.

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Sorry. Should have checked the changelog for previous updates. ... Let's just assume that "font" was a deliberate inclusion in the old entries and stick with that. Thanks!

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-4.17.0-release](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-4.17.0-release)

**Pull request merged**